src/components/InjectedComponents/ImmersiveSetup/SetupStepper.tsx

```diff
@@ -64,11 +71,10 @@ export function ImmersiveSetupStepperUI(props: ImmersiveSetupStepperUIProps) {
     const activeStep = props.currentStep
     const [, inputRef] = useCapturedInput(props.onUsernameChange)
 
-    // const backButton = (
-    //     <Button onClick={props.back} className={classes.button}>
-    //         Back
-    //     </Button>
-    // )
+    const copyText = (e: React.SyntheticEvent<HTMLElement, MouseEvent>) => {
+        window.getSelection()!.selectAllChildren(e.currentTarget)
```

Comment:
```js
// Copy to Clipboard
document.execCommand("copy");
```

Reply:
Reference Code: 
https://github.com/NiceLabs/UIKit/blob/master/src/EasyCopy.tsx

mouse enter: select element
mouse leave: deselect element
click: copy to clipboard

Reply:
I think selecting it is just enough...

Reply:
https://github.com/DimensionDev/Maskbook/blob/3fc6342858161194978caec8595cf59e276051b7/src/utils/utils.ts#L57-L63

Reply:
@SunriseFox please use `src/utils/utils.ts`#`selectElementContents` replace the line code.